### DISCUSSION TITLE
Loading Custom Fonts Locally

### DISCUSSION THREAD

**Discussion by joshgk00** [vanjarosoftware/Vanjaro.Platform#2154] [Q&A] [answered]
Is there an example of loading a custom font locally from the website? I have a site that needs a font that isn't available on Google Fonts, so I need to load it locally from the website. Is there a tool that can help generate the CSS needed for the Custom Font section in Vanjaro? What's the best way to go about that?

**Comment by joshgk00** [comment]
Bump. Is there any way to load fonts locally that aren't available as a Google Font?

**Comment by meetmandeep** [comment, marked as answer]
Sure. Just use the Custom Font option: 

![image](https://github.com/user-attachments/assets/d4114949-7626-455f-a1e2-6445a989e560)

Give your font a name, a family, and include the necessary CSS. Here is an example of CSS needed:

@font-face {
font-family: "CustomFont";
src: url("https://yoursite.com/css/fonts/CustomFont.eot");
src: url("https://yoursite.com/css/fonts/CustomFont.woff") format("woff"),
url("https://yoursite.com/css/fonts/CustomFont.otf") format("opentype"),
url("https://yoursite.com/css/fonts/CustomFont.svg#filename") format("svg");
}

Refer to this article for more details: https://www.pagecloud.com/blog/how-to-add-custom-fonts-to-any-website
